lib/date_and_time_methods.rb

```diff
@@ -7,6 +7,10 @@ def format_date_time(datetime, options = {})
     end.squish
   end
 
+  def format_date(date)
```

Comment:
Tough to know from the PHP format escape chars what this will look like - perhaps an example in a comment might be nice.

I ask because it seems like there are too many commas here, but I'm not sure